content/remote-development/getting-started.md

```diff
@@ -28,6 +28,13 @@ If the GitHub repository is not the origin of your checked out project then you
 
 ### OSX (64-bit):
 
+If you use [Homebrew](https://brew.sh/), you can install cp-remote via:
+
+```
+brew install inviqa/tools/cp-remote
```

Comment:
in the longer run, getting cp-remote into homebrew-core might be better (tho that might slow down the release process relying on homebrew devs to merge updates)